server/client_manager.py

```diff
@@ -510,15 +512,32 @@ def __init__(self, server):
         self.server = server
         self.cur_id = [i for i in range(self.server.config['playerlimit'])]
 
+
+    def new_client_auth(self,transport): #future authentication methods should go here
+        try:
+            maxclients = self.server.config['Multiclient_limit']
+        except:
+            maxclients = 3
```

Comment:
There is a convention for default config values already established in the code.

This is also not a great number to use and will cause many headaches for server owners that haven't already switched to their own tsuserver3 forks in rabid frustration at the number of issues tsuserver has.

Please raise it to 16 or 32.

Reply:
Done, didn't notice the default config values part but fixed